packages/components/src/custom-select-control-v2/item.tsx

```diff
@@ -26,4 +26,6 @@ export function CustomSelectItem( {
 	);
 }
 
+CustomSelectItem.displayName = 'CustomSelectControlV2.Item';
```

Comment:
This was necessary to ensure that React devtools and the Storybook code snippet generator displays the intended name.

Reply:
Perhaps this shouldn't be necessary if we exported them like this (untested):

```
const CustomSelectControlV2 = CustomSelect;
CustomSelectControlV2.Item = Item;
export default CustomSelectControlV2;
```

which would be something similar to what we're doing with `Tabs` right now.

Reply:
That doesn't seem to work, it looks like it just uses the name of the initial function no matter what.

And your comment made me realize that the `Tabs` subcomponents aren't showing the correct display names either 😅 I'll fix that.